lib/model/Pet.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+class Pet {
+  String rfid = '';
+  Timestamp dob = Timestamp.now();
+  String petID;
+  String name = '';
+  String type = '';
+  num lbs = 0;
+
+  Pet({this.rfid, this.dob, this.petID, this.name, this.type, this.lbs});
+
+  /* CLASS FUNCTIONS CAN GO HERE */
+
+  factory Pet.fromJson(Map<String, dynamic> parsedJson) {
+    return new Pet(
+        rfid: parsedJson['RFID'] ?? '',
+        dob: parsedJson['dob'],
+        petID: parsedJson['id'] ?? parsedJson['petID'] ?? '',
+        name: parsedJson['name'] ?? '',
+        type: parsedJson['type'] ?? '',
+        lbs: parsedJson['lbs'] ?? 0);
+  }
+
+  Map<String, dynamic> toJson() {
+    return {
+      'RFID': this.rfid,
+      'dob': this.dob,
+      'id': this.petID,
+      'name': this.name,
+      'type': this.type,
+      'lbs': this.lbs,
+    };
+  }
```

Comment:
Here I noticed you changed from using petID to id, between your getters and setters. Can we just use petID for both getter and setter to make it more readable. At first glance, I might think that id is a separate field from petID.

Reply:
I definitely agree that we should use more readable names in the json fields.

So for this I think @jlp0063 is saying we should have the json have the following fields:

- `rfid`
- `dob`
- `petID`
- `name`
- `type`
- `lbs`

In this way they map perfectly 1:1 with the fields in the actual Pet structure.